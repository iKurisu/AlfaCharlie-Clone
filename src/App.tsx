import React from "react";
import { Switch, Route, Redirect } from "react-router-dom";
import Logo from "components/Logo";
import Header from "components/Header";
import VerticalNav from "components/VerticalNav";
import MenuButton from "components/MenuButton";
import Menu from "components/Menu";
import Cursor from "components/Cursor";
import Home from "pages/Home";
import Agency from "pages/Agency";
import Work from "pages/Work";
import "./styles.scss";

const App = (): JSX.Element => (
  <React.Fragment>
    <main>
      <VerticalNav
        links={[
          ["All", true],
          ["Design", false],
          ["Wellness", false],
          ["Agency", false]
        ]}
        show={true}
      />
      <div className="scroll-content-wrapper">
        <div className="scroll-content">
          <Logo />
          <Route exact path="/" component={Header} />
          <div className="main-content">
            <Switch>
              <Route exact path="/" component={Home} />
              <Route exact path="/agency" component={Agency} />
              <Route exact path="/work" component={Work} />
              <Redirect to="/" />
            </Switch>
          </div>
        </div>
      </div>
    </main>
    <Cursor />
    <MenuButton />
    <Menu />
  </React.Fragment>
);

export default App;
