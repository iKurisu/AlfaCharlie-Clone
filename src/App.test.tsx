import React from 'react';
import App from './App';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const component: renderer.ReactTestRenderer = renderer.create(<App />)
  expect(component).toMatchSnapshot();
})