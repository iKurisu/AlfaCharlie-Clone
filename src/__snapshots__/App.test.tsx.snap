// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <main>
    <nav
      className="vertical-nav"
    >
      <ul
        className="vertical-nav-links"
      >
        <li>
          <span
            className="link-wrapper active"
          >
            <span
              className="link-name"
            >
              All
            </span>
          </span>
        </li>
        <li>
          <span
            className="link-wrapper"
          >
            <span
              className="link-name"
            >
              Agency
            </span>
          </span>
        </li>
        <li>
          <span
            className="link-wrapper"
          >
            <span
              className="link-name"
            >
              Design
            </span>
          </span>
        </li>
        <li>
          <span
            className="link-wrapper"
          >
            <span
              className="link-name"
            >
              Wellness
            </span>
          </span>
        </li>
      </ul>
    </nav>
    <div
      className="scroll-content-wrapper"
    >
      <div
        className="scroll-content"
      >
        <div
          className="logo-wrapper"
        >
          <a
            href="/"
          >
            <svg
              className="logo"
              viewBox="0 0 522.4 69.46"
              x="0"
              y="0"
            >
              <path
                d="M29.7,56.67c-0.3,0.84-0.69,1.65-1.17,2.43c-2.4,3.83-6.14,5.75-11.23,5.75c-1.68,0-3.21-0.21-4.58-0.63    c-4.67-1.5-7.01-4.91-7.01-10.24c0.06-7.85,7.79-11.92,23.18-12.22v-5.84c0-1.74-0.12-3.23-0.36-4.49    c-0.6-3.41-2.31-5.12-5.12-5.12c-0.84,0-1.59,0.15-2.25,0.45c-1.02,0.6-1.86,2.1-2.52,4.49c-0.24,0.84-0.42,1.44-0.54,1.8    c-0.9,2.52-2.64,3.77-5.21,3.77c-0.6,0-1.17-0.09-1.71-0.27c-2.22-0.6-3.32-2.07-3.32-4.4c0-1.02,0.27-1.98,0.81-2.88    c2.1-3.53,7.22-5.3,15.37-5.3c2.1,0,4.04,0.21,5.84,0.63c6.95,1.74,10.42,6.35,10.42,13.84v18.78c0,2.88,0.66,4.37,1.98,4.49    c1.74-0.06,2.79-2.16,3.14-6.29l2.34,0.09c-0.18,1.86-0.45,3.38-0.81,4.58c-1.08,3.18-3.81,4.76-8.18,4.76    C33.98,64.55,30.96,61.82,29.7,56.67z M23.41,60.72c2.52-0.96,4.34-3.53,5.48-7.73v-8.72c-2.7,0.12-4.91,0.45-6.65,0.99    c-3.23,1.2-4.85,3.77-4.85,7.73c0.06,5.21,1.59,7.88,4.58,8C22.39,60.98,22.87,60.89,23.41,60.72z"
              />
              <path
                d="M80.37,61.7c0.78,0.12,1.17,0.87,1.17,2.25H56.47v-2.34c1.68,0,3.02-0.12,4.04-0.36 c1.98-0.54,2.97-2.04,2.97-4.49V7.88h-7.01V5.55h18.42v56.07h4.49C79.74,61.61,80.07,61.64,80.37,61.7z"
              />
              <path
                d="M118.29,61.7c0.78,0.12,1.17,0.87,1.17,2.25H92.68v-2.34c4.43,0,6.71-1.56,6.83-4.67v-29.2h-7.28v-2.43h7.28    v-1.17c0-4.55,0.96-8.39,2.88-11.5c3.11-5.03,8.18-7.55,15.19-7.55c3.23,0,5.99,0.57,8.27,1.71c3.11,1.56,4.67,3.66,4.67,6.29    c0,0.54-0.09,1.05-0.27,1.53c-0.78,1.92-2.4,2.88-4.85,2.88c-1.38-0.06-2.58-0.66-3.59-1.8c-0.36-0.48-1.14-2.1-2.34-4.85    c-0.96-2.28-2.19-3.41-3.68-3.41c-1.14,0-2.07,0.42-2.79,1.26c-1.38,1.74-2.07,7.13-2.07,16.17v0.45h10.15v2.43h-10.15v33.88h6.38    C117.66,61.61,117.99,61.64,118.29,61.7z"
              />
              <path
                d="M157.35,56.67c-0.3,0.84-0.69,1.65-1.17,2.43c-2.4,3.83-6.14,5.75-11.23,5.75c-1.68,0-3.21-0.21-4.58-0.63    c-4.67-1.5-7.01-4.91-7.01-10.24c0.06-7.85,7.79-11.92,23.18-12.22v-5.84c0-1.74-0.12-3.23-0.36-4.49    c-0.6-3.41-2.31-5.12-5.12-5.12c-0.84,0-1.59,0.15-2.25,0.45c-1.02,0.6-1.86,2.1-2.52,4.49c-0.24,0.84-0.42,1.44-0.54,1.8    c-0.9,2.52-2.64,3.77-5.21,3.77c-0.6,0-1.17-0.09-1.71-0.27c-2.22-0.6-3.32-2.07-3.32-4.4c0-1.02,0.27-1.98,0.81-2.88    c2.1-3.53,7.22-5.3,15.37-5.3c2.1,0,4.04,0.21,5.84,0.63c6.95,1.74,10.42,6.35,10.42,13.84v18.78c0,2.88,0.66,4.37,1.98,4.49    c1.74-0.06,2.79-2.16,3.14-6.29l2.34,0.09c-0.18,1.86-0.45,3.38-0.81,4.58c-1.08,3.18-3.81,4.76-8.18,4.76    C161.63,64.55,158.61,61.82,157.35,56.67z M151.06,60.72c2.52-0.96,4.34-3.53,5.48-7.73v-8.72c-2.7,0.12-4.91,0.45-6.65,0.99    c-3.23,1.2-4.85,3.77-4.85,7.73c0.06,5.21,1.59,7.88,4.58,8C150.04,60.98,150.52,60.89,151.06,60.72z"
              />
              <path
                d="M212.01,30.16c3.95-4.13,9.1-6.2,15.45-6.2c4.67,0,8.57,0.93,11.68,2.79c3.17,1.98,4.76,4.28,4.76,6.92    c0,0.48-0.06,0.96-0.18,1.44c-0.72,2.28-2.61,3.41-5.66,3.41c-1.44,0-2.58-0.3-3.41-0.9c-1.08-0.78-1.8-2.55-2.16-5.3    c-0.18-1.14-0.33-1.95-0.45-2.43c-0.6-2.46-2.22-3.68-4.85-3.68c-1.32,0-2.46,0.27-3.41,0.81c-3.18,1.92-4.76,7.91-4.76,17.97    c0,5.03,0.51,8.93,1.53,11.68c1.38,3.71,4.13,5.57,8.27,5.57c1.56,0,3.11-0.24,4.67-0.72c4.01-1.26,6.86-3.68,8.54-7.28l2.34,0.63   c-0.6,1.56-1.47,3-2.61,4.31c-3.36,3.77-8.15,5.66-14.38,5.66c-4.01,0-7.58-0.75-10.69-2.25c-6.65-3.29-9.97-9.31-9.97-18.06    C206.71,38.67,208.47,33.88,212.01,30.16z"
              />
              <path
                d="M274.9,61.7c0.78,0.12,1.17,0.87,1.17,2.25h-21.12v-2.34c0.42,0,0.84-0.03,1.26-0.09    c2.7-0.24,4.04-1.77,4.04-4.58V7.88h-5.3V5.55h16.71v26.51c0.42-0.84,0.93-1.68,1.53-2.51c2.88-3.71,6.74-5.57,11.59-5.57    c1.56,0,2.99,0.24,4.31,0.72c5.33,1.8,8,6.44,8,13.93v23h2.97c0.36,0,0.72,0.03,1.08,0.09c0.78,0.12,1.17,0.87,1.17,2.25h-20.13    v-2.34c1.02,0,1.83-0.15,2.43-0.45c0.72-0.42,1.08-1.83,1.08-4.22V37.08c0-0.6-0.03-1.14-0.09-1.62c-0.42-4.49-2.46-6.74-6.11-6.74    c-0.96,0-1.95,0.27-2.97,0.81c-3.23,1.74-4.85,5.24-4.85,10.51v21.56h2.25C274.27,61.61,274.6,61.64,274.9,61.7z"
              />
              <path
                d="M337.12,56.67c-0.3,0.84-0.69,1.65-1.17,2.43c-2.4,3.83-6.14,5.75-11.23,5.75c-1.68,0-3.21-0.21-4.58-0.63    c-4.67-1.5-7.01-4.91-7.01-10.24c0.06-7.85,7.79-11.92,23.18-12.22v-5.84c0-1.74-0.12-3.23-0.36-4.49    c-0.6-3.41-2.31-5.12-5.12-5.12c-0.84,0-1.59,0.15-2.25,0.45c-1.02,0.6-1.86,2.1-2.52,4.49c-0.24,0.84-0.42,1.44-0.54,1.8    c-0.9,2.52-2.64,3.77-5.21,3.77c-0.6,0-1.17-0.09-1.71-0.27c-2.22-0.6-3.32-2.07-3.32-4.4c0-1.02,0.27-1.98,0.81-2.88    c2.1-3.53,7.22-5.3,15.37-5.3c2.1,0,4.04,0.21,5.84,0.63c6.95,1.74,10.42,6.35,10.42,13.84v18.78c0,2.88,0.66,4.37,1.98,4.49    c1.74-0.06,2.79-2.16,3.14-6.29l2.34,0.09c-0.18,1.86-0.45,3.38-0.81,4.58c-1.08,3.18-3.8,4.76-8.18,4.76    C341.4,64.55,338.38,61.82,337.12,56.67z M330.83,60.72c2.52-0.96,4.34-3.53,5.48-7.73v-8.72c-2.69,0.12-4.91,0.45-6.65,0.99    c-3.24,1.2-4.85,3.77-4.85,7.73c0.06,5.21,1.59,7.88,4.58,8C329.81,60.98,330.29,60.89,330.83,60.72z"
              />
              <path
                d="M389.28,61.7c0.78,0.12,1.17,0.87,1.17,2.25h-25.43v-2.34c0.78,0,1.47-0.06,2.07-0.18    c2.1-0.36,3.15-1.86,3.15-4.49V27.65h-5.21v-2.34h15.72l0.63,8.36c0.66-1.68,1.62-3.23,2.88-4.67c2.81-2.93,6.08-4.4,9.79-4.4    c0.78,0,1.53,0.09,2.25,0.27c2.93,0.78,4.4,2.67,4.4,5.66c0,0.84-0.18,1.65-0.54,2.43c-1.02,2.28-2.73,3.41-5.12,3.41    c-0.72,0-1.38-0.09-1.98-0.27c-1.26-0.48-2.43-1.68-3.5-3.59c-0.78-1.38-1.5-2.07-2.16-2.07c-0.36,0-0.75,0.09-1.17,0.27    c-2.64,1.38-4.16,5.21-4.58,11.5v19.41h6.65C388.65,61.61,388.98,61.64,389.28,61.7z"
              />
              <path
                d="M430.54,61.7c0.78,0.12,1.17,0.87,1.17,2.25h-25.07v-2.34c1.68,0,3.02-0.12,4.04-0.36    c1.98-0.54,2.97-2.04,2.97-4.49V7.88h-7.01V5.55h18.42v56.07h4.49C429.91,61.61,430.24,61.64,430.54,61.7z"
              />
              <path
                d="M466.94,61.7c0.78,0.12,1.17,0.87,1.17,2.25H446v-2.34c0.78,0,1.47-0.06,2.07-0.18    c2.1-0.36,3.14-1.86,3.14-4.49V27.65H446v-2.34h16.62v36.3h3.32C466.31,61.61,466.64,61.64,466.94,61.7z M462.53,16.78    c-1.26,2.22-3.2,3.32-5.84,3.32c-1.08,0-2.13-0.27-3.14-0.81c-2.34-1.2-3.5-3.11-3.5-5.75c0-1.14,0.27-2.21,0.81-3.23    c1.32-2.34,3.26-3.5,5.84-3.5c1.14,0,2.22,0.3,3.24,0.9c2.34,1.26,3.5,3.21,3.5,5.84C463.43,14.68,463.13,15.76,462.53,16.78z"
              />
              <path
                d="M515.28,57.48c-3.06,4.91-8.09,7.37-15.1,7.37c-3.95,0-7.49-0.81-10.6-2.43c-6.53-3.41-9.79-9.46-9.79-18.15    c0-5.33,1.44-9.79,4.31-13.39c3.77-4.61,9.05-6.92,15.81-6.92c3.71,0,7.04,0.63,9.97,1.89c5.21,2.34,7.82,6.05,7.82,11.14    c0,1.02-0.18,1.95-0.54,2.79c-1.56,3.36-7.16,4.88-16.8,4.58l-8.27-0.27c0.06,3.47,0.36,6.47,0.9,8.98    c1.38,6.11,4.46,9.16,9.25,9.16c1.5,0,2.97-0.27,4.4-0.81c3.59-1.32,6.23-3.8,7.91-7.46l2.16,0.72    C516.36,55.65,515.88,56.58,515.28,57.48z M492.1,41.49l5.75,0.27c2.4,0.12,4.19-0.03,5.39-0.45c1.68-0.66,2.52-2.4,2.52-5.21    c-0.06-6.41-2.01-9.67-5.84-9.79C495.12,26.48,492.52,31.54,492.1,41.49z"
              />
            </svg>
          </a>
        </div>
        <div
          className="header"
        >
          <h1>
            SAN DIEGO CREATIVE AGENCY
          </h1>
        </div>
        <div
          className="main-content"
        />
      </div>
    </div>
  </main>,
  <div
    className="menu-button"
    onClick={[Function]}
  >
    <svg
      className="menu-hamburger"
      viewBox="0 0 72 72"
      x="0"
      y="0"
    >
      <line
        x1="2"
        x2="69"
        y1="9"
        y2="9"
      />
      <line
        x1="2"
        x2="69"
        y1="35"
        y2="35"
      />
      <line
        x1="2"
        x2="69"
        y1="61"
        y2="61"
      />
    </svg>
    <svg
      className="menu-cross"
      viewBox="0 0 59 59"
      x="0px"
      y="0px"
    >
      <line
        x1="6.7"
        x2="53"
        y1="6.5"
        y2="53"
      />
      <line
        x1="6.7"
        x2="53"
        y1="53"
        y2="6.5"
      />
    </svg>
  </div>,
  <div
    className="menu hide"
  >
    <div
      className="menu-nav-wrapper -flex"
    >
      <a
        className="menu-home-link"
      >
        <svg
          className="menu-logo"
          viewBox="0 0 156.23 156.23"
          x="0px"
          y="0px"
        >
          <path
            d="M34.8,82.75c0.25,0.53,0.44,1.08,0.58,1.66c0.66,2.87-0.18,5.48-2.53,7.83c-0.77,0.77-1.57,1.38-2.4,1.82   c-2.84,1.47-5.5,0.97-7.96-1.49c-3.59-3.65-1.91-9.09,5.06-16.33l-2.69-2.69c-0.8-0.8-1.55-1.44-2.24-1.91   c-1.85-1.3-3.43-1.3-4.72,0c-0.39,0.39-0.66,0.8-0.83,1.24c-0.19,0.75,0.11,1.82,0.91,3.23c0.28,0.5,0.47,0.86,0.58,1.08   c0.75,1.57,0.52,2.96-0.66,4.14c-0.28,0.28-0.58,0.5-0.91,0.66c-1.3,0.75-2.49,0.58-3.56-0.5c-0.47-0.47-0.79-1.04-0.95-1.7   c-0.66-2.6,0.88-5.77,4.64-9.53c0.97-0.97,1.96-1.77,2.98-2.4c4.01-2.4,7.74-1.88,11.19,1.57l8.66,8.66   c1.33,1.33,2.32,1.71,2.98,1.16c0.77-0.83,0.29-2.28-1.45-4.35l1.12-1.04c0.77,0.94,1.35,1.77,1.74,2.49   c0.97,1.96,0.44,3.95-1.57,5.97C40.41,84.41,37.76,84.55,34.8,82.75z M33.76,87.52c0.72-1.6,0.37-3.63-1.04-6.09l-4.02-4.02   c-1.19,1.3-2.06,2.47-2.61,3.52c-0.94,2.05-0.5,3.98,1.33,5.8c2.43,2.38,4.36,2.9,5.8,1.57C33.42,88.11,33.6,87.85,33.76,87.52z"
          />
          <path
            d="M66.67,26.43c-0.08-3.73,1.34-7.06,4.27-9.99c2.15-2.15,4.38-3.52,6.67-4.1c2.38-0.55,4.17-0.22,5.39,0.99   c0.22,0.22,0.41,0.47,0.58,0.75c0.72,1.38,0.37,2.78-1.04,4.19c-0.66,0.66-1.33,1.05-1.99,1.16c-0.86,0.14-2-0.35-3.44-1.45   c-0.61-0.44-1.05-0.75-1.33-0.91c-1.41-0.86-2.72-0.68-3.94,0.54c-0.61,0.61-1.01,1.26-1.2,1.95c-0.58,2.35,1.45,5.84,6.09,10.48   c2.32,2.32,4.35,3.88,6.09,4.68c2.35,1.08,4.47,0.66,6.38-1.24c0.72-0.72,1.33-1.55,1.82-2.49c1.27-2.43,1.46-4.86,0.58-7.29   l1.37-0.79c0.44,0.99,0.7,2.06,0.79,3.19c0.19,3.29-1.15,6.37-4.02,9.24c-1.85,1.85-3.84,3.15-5.97,3.9   c-4.59,1.55-8.89,0.3-12.93-3.73C68.15,32.8,66.75,29.77,66.67,26.43z"
          />
          <path
            d="M121.43,73.49c-0.25-0.53-0.44-1.08-0.58-1.66c-0.66-2.87,0.18-5.48,2.53-7.83c0.77-0.77,1.57-1.38,2.4-1.82   c2.84-1.47,5.5-0.97,7.96,1.49c3.59,3.65,1.91,9.09-5.06,16.33l2.69,2.69c0.8,0.8,1.55,1.44,2.24,1.91c1.85,1.3,3.43,1.3,4.72,0   c0.39-0.39,0.66-0.8,0.83-1.24c0.19-0.75-0.11-1.82-0.91-3.23c-0.28-0.5-0.47-0.86-0.58-1.08c-0.75-1.57-0.52-2.96,0.66-4.14   c0.28-0.28,0.58-0.5,0.91-0.66c1.3-0.75,2.49-0.58,3.56,0.5c0.47,0.47,0.79,1.04,0.95,1.7c0.66,2.6-0.88,5.77-4.64,9.53   c-0.97,0.97-1.96,1.77-2.98,2.4c-4.01,2.4-7.74,1.88-11.19-1.57l-8.66-8.66c-1.33-1.33-2.32-1.71-2.98-1.16   c-0.77,0.83-0.29,2.28,1.45,4.35l-1.12,1.04c-0.77-0.94-1.35-1.77-1.74-2.49c-0.97-1.96-0.44-3.95,1.57-5.97   C115.83,71.83,118.48,71.69,121.43,73.49z M122.47,68.72c-0.72,1.6-0.37,3.63,1.04,6.09l4.02,4.02c1.19-1.3,2.06-2.47,2.61-3.52   c0.94-2.05,0.5-3.98-1.33-5.8c-2.43-2.38-4.36-2.9-5.8-1.57C122.82,68.13,122.64,68.39,122.47,68.72z"
          />
          <path
            d="M89.56,129.8c0.08,3.73-1.34,7.06-4.27,9.99c-2.15,2.15-4.38,3.52-6.67,4.1c-2.38,0.55-4.17,0.22-5.39-0.99   c-0.22-0.22-0.41-0.47-0.58-0.75c-0.72-1.38-0.37-2.78,1.04-4.19c0.66-0.66,1.33-1.05,1.99-1.16c0.86-0.14,2,0.35,3.44,1.45   c0.61,0.44,1.05,0.75,1.33,0.91c1.41,0.86,2.72,0.68,3.94-0.54c0.61-0.61,1.01-1.26,1.2-1.95c0.58-2.35-1.45-5.84-6.09-10.48   c-2.32-2.32-4.35-3.88-6.09-4.68c-2.35-1.08-4.47-0.66-6.38,1.24c-0.72,0.72-1.33,1.55-1.82,2.49c-1.27,2.43-1.46,4.86-0.58,7.29   l-1.37,0.79c-0.44-0.99-0.7-2.06-0.79-3.19c-0.19-3.29,1.15-6.37,4.02-9.24c1.85-1.85,3.84-3.15,5.97-3.9   c4.59-1.55,8.89-0.3,12.93,3.73C88.08,123.44,89.48,126.46,89.56,129.8z"
          />
          <g
            className="menu-logo-lines"
          >
            <line
              x1="28.39"
              x2="127.84"
              y1="28.39"
              y2="127.84"
            />
            <line
              x1="28.39"
              x2="127.84"
              y1="127.84"
              y2="28.39"
            />
          </g>
        </svg>
      </a>
      <nav
        className="menu-nav-links -flex"
        onMouseLeave={[Function]}
        style={
          Object {
            "opacity": 0,
          }
        }
      >
        <a
          className="menu-nav-link"
          onMouseEnter={[Function]}
        >
          <span
            className="link-characters"
          >
            <span
              className="link-character"
              style={
                Object {
                  "opacity": 0,
                }
              }
            >
              a
            </span>
            <span
              className="link-character"
              style={
                Object {
                  "opacity": 0,
                }
              }
            >
              g
            </span>
            <span
              className="link-character"
              style={
                Object {
                  "opacity": 0,
                }
              }
            >
              e
            </span>
            <span
              className="link-character"
              style={
                Object {
                  "opacity": 0,
                }
              }
            >
              n
            </span>
            <span
              className="link-character"
              style={
                Object {
                  "opacity": 0,
                }
              }
            >
              c
            </span>
            <span
              className="link-character"
              style={
                Object {
                  "opacity": 0,
                }
              }
            >
              y
            </span>
          </span>
        </a>
        <a
          className="menu-nav-link"
          onMouseEnter={[Function]}
        >
          <span
            className="link-characters"
          >
            <span
              className="link-character"
              style={
                Object {
                  "opacity": 0,
                }
              }
            >
              w
            </span>
            <span
              className="link-character"
              style={
                Object {
                  "opacity": 0,
                }
              }
            >
              o
            </span>
            <span
              className="link-character"
              style={
                Object {
                  "opacity": 0,
                }
              }
            >
              r
            </span>
            <span
              className="link-character"
              style={
                Object {
                  "opacity": 0,
                }
              }
            >
              k
            </span>
          </span>
        </a>
        <a
          className="menu-nav-link"
          onMouseEnter={[Function]}
        >
          <span
            className="link-characters"
          >
            <span
              className="link-character"
              style={
                Object {
                  "opacity": 0,
                }
              }
            >
              j
            </span>
            <span
              className="link-character"
              style={
                Object {
                  "opacity": 0,
                }
              }
            >
              o
            </span>
            <span
              className="link-character"
              style={
                Object {
                  "opacity": 0,
                }
              }
            >
              u
            </span>
            <span
              className="link-character"
              style={
                Object {
                  "opacity": 0,
                }
              }
            >
              r
            </span>
            <span
              className="link-character"
              style={
                Object {
                  "opacity": 0,
                }
              }
            >
              n
            </span>
            <span
              className="link-character"
              style={
                Object {
                  "opacity": 0,
                }
              }
            >
              a
            </span>
            <span
              className="link-character"
              style={
                Object {
                  "opacity": 0,
                }
              }
            >
              l
            </span>
          </span>
        </a>
        <a
          className="menu-nav-link"
          onMouseEnter={[Function]}
        >
          <span
            className="link-characters"
          >
            <span
              className="link-character"
              style={
                Object {
                  "opacity": 0,
                }
              }
            >
              c
            </span>
            <span
              className="link-character"
              style={
                Object {
                  "opacity": 0,
                }
              }
            >
              o
            </span>
            <span
              className="link-character"
              style={
                Object {
                  "opacity": 0,
                }
              }
            >
              n
            </span>
            <span
              className="link-character"
              style={
                Object {
                  "opacity": 0,
                }
              }
            >
              t
            </span>
            <span
              className="link-character"
              style={
                Object {
                  "opacity": 0,
                }
              }
            >
              a
            </span>
            <span
              className="link-character"
              style={
                Object {
                  "opacity": 0,
                }
              }
            >
              c
            </span>
            <span
              className="link-character"
              style={
                Object {
                  "opacity": 0,
                }
              }
            >
              t
            </span>
          </span>
        </a>
      </nav>
    </div>
    <div
      className="menu-images -flex"
    >
      <div
        className="menu-slider"
      >
        <div
          className="slider-wrapper"
          style={
            Object {
              "transform": "translateX(0px)",
              "transition": "transform 2s",
            }
          }
        >
          <div
            className="slide"
          >
            <div
              className="slide-img"
              style={
                Object {
                  "backgroundImage": "url(https://alfacharlie.b-cdn.net/wp-content/uploads/2019/05/Alfa-Charlie-Creative-Agency-home-e1558112927714.jpg)",
                  "transform": "translateX(1px)",
                  "transition": "transform 2s",
                }
              }
            />
          </div>
          <div
            className="slide"
          >
            <div
              className="slide-img"
              style={
                Object {
                  "backgroundImage": "url(https://alfacharlie.b-cdn.net/wp-content/uploads/2019/05/Alfa-Charlie-Creative-Agency-about-5.jpg)",
                  "transform": "translateX(-338px)",
                  "transition": "transform 2s",
                }
              }
            />
          </div>
          <div
            className="slide"
          >
            <div
              className="slide-img"
              style={
                Object {
                  "backgroundImage": "url(https://alfacharlie.b-cdn.net/wp-content/uploads/2019/04/Alfa-Charlie-Creative-Agency-work.jpg)",
                  "transform": "translateX(-677px)",
                  "transition": "transform 2s",
                }
              }
            />
          </div>
          <div
            className="slide"
          >
            <div
              className="slide-img"
              style={
                Object {
                  "backgroundImage": "url(https://alfacharlie.b-cdn.net/wp-content/uploads/2019/05/Alfa-Charlie-Creative-Agency-home-2.jpg)",
                  "transform": "translateX(-1016px)",
                  "transition": "transform 2s",
                }
              }
            />
          </div>
          <div
            className="slide"
          >
            <div
              className="slide-img"
              style={
                Object {
                  "backgroundImage": "url(https://alfacharlie.b-cdn.net/wp-content/uploads/2019/05/Alfa-Charlie-Creative-Agency-contact-2.jpg)",
                  "transform": "translateX(-1355px)",
                  "transition": "transform 2s",
                }
              }
            />
          </div>
        </div>
      </div>
    </div>
    <nav
      className="vertical-nav"
    >
      <ul
        className="vertical-nav-links"
      >
        <li>
          <span
            className="link-wrapper"
          >
            <span
              className="link-name"
            >
              Instagram
            </span>
          </span>
        </li>
        <li>
          <span
            className="link-wrapper"
          >
            <span
              className="link-name"
            >
              Privacy
            </span>
          </span>
        </li>
      </ul>
    </nav>
    <div
      className="menu-white"
      style={
        Object {
          "transform": "translateX(100%)",
        }
      }
    />
    <div
      className="menu-gray"
      style={
        Object {
          "transform": "translateX(-100%)",
        }
      }
    />
  </div>,
]
`;
